ote/src/cljs/ote/views/transit_visualization/change_icons.cljs

```diff
@@ -38,6 +39,17 @@
       changes
       (when with-labels? " aikataulumuutosta")]]))
 
+(defn no-traffic-icons [{:keys [change-type] :as change-summary}]
```

Comment:
Is `no-traffic-icons` referred to anywhere?

Reply:
Yes. in transit-visualization.cljs on row 403

Reply:
Instead of `{:flex "0.5"}` the implicit flex-grow number could be `{:flex "1"}` because it's the ratio that matters.
But this is nitpicking.

Reply:
All the flex numbers could be changed to integers with the proper ratios. I don't think floats are all that good for flex.